sdk/Trace/IdGenerator.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Sdk\Trace;
+
+interface IdGenerator
+{
+    public function generateTraceId(): string;
```

Comment:
In the spec, these are labeled `generateSpanIdBytes` and `generateTraceIdBytes`  - should we follow suit with the spec here?

Reply:
In the spec they show an example from Java implementation, where they also have a dedicated classes `TraceId` and `SpanId` for this. I don't think we have `bytes` here in PHP yet, for now Trace ID and Span ID are plain strings. In such case more suitable name is `generateTraceIdString` which sounds too obvious.